static/templates/invite_user.hbs

```diff
@@ -30,6 +30,19 @@
                     </div>
                 </div>
                 <div class="input-group">
+                    <label for="expires_in">{{t "Invitation expires after" }}</label>
+                    <div>
+                        <select id="expires_in">
+                            {{!-- <option name="expires_in" value="{{ expires_in_options.hour.value }}">{{t "1 hour"}}</option> --}}
+                            <option name="expires_in" value="{{ expires_in_options.day.value }}">{{t "1 day"}}</option>
+                            <option name="expires_in" value="{{ expires_in_options.threeDays.value }}" >{{t "3 days"}}</option>
+                            <option name="expires_in" value="{{  expires_in_options.tenDays.value }}" selected="selected" >{{t "10 days"}}</option>
+                            <option name="expires_in" value="{{ expires_in_options.thirtyDays.value }}">{{t "30 days"}}</option>
+                            {{!-- <option name="expires_in" value="{{ expires_in_options.never.value }}">{{t "Never expires"}}</option> --}}
```

Comment:
I think it'd be better to have the display durations come from the `settings_config` data structure; if you did that, this code here could just be a loop over those values.

Reply:
Yes! Good suggestion, I updated it to a loop that solves this.